### PR TITLE
Suggestions for GraphQL PR

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
### Checklist

- [x] Appropriate branch selected => this is a suggestion change for another PR
- [x] Any modified or new methods or classes have helpful JavaDoc and code is thoroughly commented
- [na] The description lists all applicable issues this PR seeks to resolve
- [na] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

Some suggestions to make logging of trip requests work with #219 (Coordination needed for nomenclature and duplicate GraphQL structure classes).

### CODE REVIEW

File: src/main/java/org/opentripplanner/middleware/otp/OtpGraphQLTransportMode.java
Comment (JymDyerIBI) [inline src/main/java/org/opentripplanner/middleware/otp/OtpGraphQLTransportMode.java:1]: This looks good. I take it this is the common transportMode and not all the fancier sub-modes in the UI?

**Review by JymDyerIBI — APPROVED** [review]
Definitely looks good to me.

File: src/main/java/org/opentripplanner/middleware/otp/OtpGraphQLTransportMode.java
Comment (binh-dam-ibigroup) [reply src/main/java/org/opentripplanner/middleware/otp/OtpGraphQLTransportMode.java:1]: Yes, this is the data structure that gets passed in the GraphQL variables for the modes, such as `{ mode: "BICYCLE", qualifier: "RENT"  }`.

**Review by JymDyerIBI — APPROVED** [review]
This new batch of changes looks good.

**Review by JymDyerIBI — APPROVED** [review]
👍  LGTM!

**Comment by binh-dam-ibigroup** [comment]
Closing in favor of #245.

**Pull request closed without merging**